docs/content/node-operation/faq.mdx

```diff
@@ -27,3 +27,23 @@ No. The node addresses are baked-in as part of the identity table stored on-chai
 ### Can I bootstrap and run a node at anytime?
 
 The node has to be included as part of the spork and only then will be part of the Flow protocol. Hence, a node can be bootstrapped only during a spork.
+
+### Would it hurt the network to have a node that constantly spins up and down?
+
+A node needs to be part of the genesis block. The node can bounce after that and while that may not break the Flow protocol, it will cause severe degradation of overall performance.
+A way to prevent this is to check your equipment meets Flow's recommended requirements, periodically checking for updates and announcements in Discord but also using a node monitoring system for when your node does go offline.
```

Comment:
"A way to prevent this is to check your equipment meets Flow's recommended requirements..."
Could you add a link to the information about requirements?